apps/src/code-studio/progress.js

```diff
@@ -359,6 +365,7 @@ function initializeStoreWithProgress(
       isFullProgress: isFullProgress,
       isLessonExtras: isLessonExtras,
       currentPageNumber: currentPageNumber,
+      currentSublevelPosition: currentSublevelPosition,
```

Comment:
non-blocking, but could we crunch these down to the shorthand properties where possible?

e.g., `currentSublevelPosition` instead of `currentSublevelPosition : currentSublevelPosition`.

I think there's a linter rule we could enable to do that automatically as well...

Reply:
Oh yeah, I kept it the same for consistency but would be nice to use shorthands for these. Will take a look